File: generated-sources/php-ze-ph/mojang-api/src/App/com/github/asyncmc/mojang/api/php/ze/ph/model/SecurityAnswerId.php
<?php

namespace com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 * A security question answer id given by Mojang
 */
class SecurityAnswerId
{
    /**
     * A number giver by Mojang to reply a question
     * @DTA\Data(field="id")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $id;
}

File: generated-sources/kotlin-spring/mojang-api/src/main/kotlin/com/github/asyncmc/mojang/api/kotlin/spring/model/SecurityAnswerId.kt
package com.github.asyncmc.mojang.api.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * A security question answer id given by Mojang
 * @param id A number giver by Mojang to reply a question
 */
data class SecurityAnswerId (

        @get:NotNull 
        @JsonProperty("id") val id: Int
) {

}

